### PR TITLE
Use the String component

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: src/Twig/SourceCodeExtension.php [hunk 1]
```diff
@@ -117,20 +118,20 @@ private function getTemplateSource($template): array
...
     private function unindentCode(string $code): string
     {
-        $formattedCode = $code;
-        $codeLines = explode("\n", $code);
+        $codeLines = u($code)->split("\n");
 
-        $indentedLines = array_filter($codeLines, function ($lineOfCode) {
-            return '' === $lineOfCode || 0 === mb_strpos($lineOfCode, '    ');
+        $indentedOrBlankLines = array_filter($codeLines, function ($lineOfCode) {
+            return u($lineOfCode)->isEmpty() || u($lineOfCode)->startsWith('    ');
```
Comment (stof) [inline src/Twig/SourceCodeExtension.php]: you should probably instantiate only one string object and reuse it.

Comment (javiereguiluz) [reply src/Twig/SourceCodeExtension.php]: For an application where performance is critical, I'd agree ... but this is only used when checking the source code explicitly, so it rarely happens. Let's keep the code a bit simpler. Thanks.

File: .travis.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ cache:
 env:
   global:
     - SYMFONY_PHPUNIT_DIR=./bin/.phpunit
-    - SYMFONY_DEPRECATIONS_HELPER=0
+    - SYMFONY_DEPRECATIONS_HELPER=9
```
Comment (COil) [inline .travis.yml]: Hi, @javiereguiluz  What is 9 ? :thinking:

Comment (javiereguiluz) [reply .travis.yml]: They are the Doctrine deprecations that I couldn't remove and prevented tests from being green. See https://travis-ci.org/symfony/demo/jobs/627625205#L358

**Pull request merged**